shaders/annotation.glsl

```diff
@@ -0,0 +1,47 @@
+#version 150
+// Copyright 2015, Christopher J. Foster and the other displaz contributors.
+// Use of this code is governed by the BSD-style license found in LICENSE.txt
+
+// Draws annotations, which are textured quads that always face the camera,
+// arent't obscured by other objects, and don't shrink when they're further
+// away.
+
+uniform mat4 modelViewMatrix;
+uniform mat4 projectionMatrix;
+uniform mat4 modelViewProjectionMatrix;
+uniform vec2 annotationSize; // In pixels
+uniform vec2 viewportSize; // In pixels
+
+//------------------------------------------------------------------------------
+#if defined(VERTEX_SHADER)
+
+in vec2 position;
```

Comment:
A couple of comments about the coordinate system of `position` would be helpful here or down below: `makeVAO`, `Annotation::draw()` and the shader work together in a tightly coupled way, and it's just a little tricky to figure out what's going on.